dra-evolution/testdata/pod-one-container-one-gpu-one-vf.yaml

```diff
@@ -63,17 +63,23 @@ spec:
     - deviceSelector: "device.stringAttributes['sriov-nic.example.org/sriovType'] == 'vf'"
 status:
   allocation:
-    nodeName: worker-1
+    availableOnNodes:
```

Comment:
I'm confused. Why is this here, if this is not an example of a classic DRA allocation? It should not be needed except for classic DRA, right?

Reply:
I am preparing `resourceClaim.status` for network-attached devices here. We didn't have those at all before. Now we get support for them through classic DRA and eventually for structured parameters when we get to https://github.com/kubernetes-sigs/wg-device-management/issues/21.

Reply:
I simplified this field in https://github.com/kubernetes-sigs/wg-device-management/pull/14 because I wanted to avoid the distraction (minimal API first, extensions later). This is one of those extensions.